endpoint.bal

```diff
@@ -0,0 +1,536 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/url;
+
+# Twitter Client object.
+#
+# + clientId - The consumer key of the Twitter account
+# + clientSecret - The consumer secret of the Twitter account
+# + accessToken - The access token of the Twitter account
+# + accessTokenSecret - The access token secret of the Twitter account
+# + twitterClient - HTTP Client endpoint
+@display {label: "Twitter API Client"}
```

Comment:
Going by today's conventions this should be `Twitter`

Reply:
Ack. Fixed